### PR TITLE
Run builds in parallel

### PR DESCRIPTION
**Pull request by nhhagen**

### CODE REVIEW

File: images/benthos-v4/context/Dockerfile [hunk 1]
```diff
@@ -1 +1,3 @@
 FROM docker.io/jeffail/benthos:4@sha256:5348382a06806c947e228cf111e71446afd4220a7486644482a741ac0545ac59
+
+RUN echo "haha"
```
Comment (aucampia) [inline images/benthos-v4/context/Dockerfile:3]: is this needed? If it is maybe rather go for `true`

File: images/poetry-python3.8/context/Dockerfile [hunk 2]
```diff
@@ -37,3 +37,5 @@ RUN python3 -m venv $POETRY_HOME && \
         --use-pep517 \
         --requirement requirements-poetry.txt && \
     rm -rf requirements-poetry.txt
+
+RUN echo "haha"
```
Comment (aucampia) [inline images/poetry-python3.8/context/Dockerfile:41]: is this needed? If it is maybe rather go for `true`

Comment (nhhagen) [reply images/poetry-python3.8/context/Dockerfile:41]: You beat me to my comment, I'll remove the `echo` calls before merging. Needed to trigger changes in the actual images.

**Review by aucampia — DISMISSED** [review]
Looks good

**Comment by nhhagen** [comment]
> Looks good

thx, can you reapprove, I remove the `echo` statements.

**Comment by nhhagen** [comment]
> > Looks good
> 
> thx, can you reapprove, I remove the `echo` statements.

Never mind, need to account for no changed files.

**Comment by nhhagen** [comment]
@aucampia fixed for no image changes as well.

**Pull request merged**